app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,67 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def new #only cares about showing the form
+    @task = Task.new
+  end
+
+  def create # has access to user data, from the form
+    task = Task.new(name: params[:task][:name], description: params[:task][:description], completion_date: params[:task][:completion_date])
+
+    # task.name = params[:task][:name]
+
+    task.save
+    redirect_to('/tasks')
```

Comment:
Ideally here we would use `redirect_to tasks_path` instead. It should work the same way as this line does, but is more flexible if we for some reason need to change the URL that maps to the `Tasks#index` action.